app/layout/aside_feed.phtml

```diff
@@ -8,8 +8,12 @@
 	}
 
 	$state_filter = FreshRSS_Context::$user_conf->show_fav_unread ? '&state=3' : '';
```

Comment:
At a quick glance, you're sure you shouldn't use this variable to do the honors? :-)